static/bidder-info/inmobi.yaml

```diff
@@ -12,8 +12,9 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
 userSync:
-  redirect:
-    url: "https://sync.inmobi.com/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}"
+  iframe:
+    url: "https://sync.inmobi.com/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
     userMacro: "{ID5UID}"
```

Comment:
redirection is failing 

```
https://sync.inmobi.com/prebid?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dinmobi%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%7BID5UID%7D%22
```

Reply:
Hi @onkarvhanumante , the URL used by you seems to have "" before user macro 
i.e. - https://prebid.adnxs.com/pbs/v1/setuid?bidder=inmobi&gdpr=&gdpr_consent=&f=i&uid="{ID5UID}"
Whereas, it should be without macro 
i.e. - https://prebid.adnxs.com/pbs/v1/setuid?bidder=inmobi&gdpr=&gdpr_consent=&f=i&uid={ID5UID}

Correcting the same URL as you used - 
https://sync.inmobi.com/prebid?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dinmobi%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%7BID5UID%7D

Can observe redirection happening successfully, as share in screenshot
<img width="1728" alt="Screenshot 2024-05-23 at 18 13 40" src="https://github.com/prebid/prebid-server/assets/7847210/4fbcf611-a923-483c-b8c2-57e6d2140df9">

Kindly help, Many Thanks!

Reply:
URl seems to be working now. Thanks!

Reply:
> Hi @onkarvhanumante , the URL used by you seems to have "" before user macro i.e. - [https://prebid.adnxs.com/pbs/v1/setuid?bidder=inmobi&gdpr=&gdpr_consent=&f=i&uid="{ID5UID}](https://prebid.adnxs.com/pbs/v1/setuid?bidder=inmobi&gdpr=&gdpr_consent=&f=i&uid=%22%7BID5UID%7D)" Whereas, it should be without macro i.e. - [https://prebid.adnxs.com/pbs/v1/setuid?bidder=inmobi&gdpr=&gdpr_consent=&f=i&uid={ID5UID}](https://prebid.adnxs.com/pbs/v1/setuid?bidder=inmobi&gdpr=&gdpr_consent=&f=i&uid=%7BID5UID%7D)
> 
> Correcting the same URL as you used - https://sync.inmobi.com/prebid?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dinmobi%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%7BID5UID%7D
> 
> Can observe redirection happening successfully, as share in screenshot <img alt="Screenshot 2024-05-23 at 18 13 40" width="1728" src="https://private-user-images.githubusercontent.com/7847210/333187104-4fbcf611-a923-483c-b8c2-57e6d2140df9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3OTA1NTgsIm5iZiI6MTcxNjc5MDI1OCwicGF0aCI6Ii83ODQ3MjEwLzMzMzE4NzEwNC00ZmJjZjYxMS1hOTIzLTQ4M2MtYjhjMi01N2U2ZDIxNDBkZjkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjdUMDYxMDU4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzdkOWQyYjM5YjA1ZTdhNGE5YjYzY2ZkZjcwZjE4YjdkODM1YWM1ODNjOTBmMGQ4MDgwZDVmNDA4YTE1YTY1ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.bk20wnc86a0J8KOpfyh_JNVC9ktaYS_08RrxCJw9FjY">
> 
> Kindly help, Many Thanks!

@SiddhantAgrawal  should remove double quotes from  usermacro vale `"{ID5UID}"`

Reply:
> The final URL that we honour should not have doublequotes, and after removing doublequotes the redirection is working as pasted screenshot above. Kindly help, thanks

suggesting to change `userMacro: "{ID5UID}"` to `userMacro: {ID5UID}`

Reply:
Hi @onkarvhanumante, we have not seen any cookie sync API requests coming to our controller having "" in the userMacro. Not sure, if removing quotes is right thing to do. I read the datatype of the key as [String](https://github.com/prebid/prebid-server/blob/e13d786d5ec2db1ffd2206f25546f2f41ed62e78/config/bidderinfo.go#L197)
Kindly help, thanks.
cc @gargcreation1992

Reply:
Hi @onkarvhanumante , could you please check?